cpp/include/cudf/detail/scan.hpp

```diff
@@ -102,5 +102,17 @@ std::unique_ptr<column> inclusive_dense_rank_scan(column_view const& order_by,
                                                   rmm::cuda_stream_view stream,
                                                   rmm::mr::device_memory_resource* mr);
 
+/**
+ * @brief Generate row percent ranks for a column.
+ *
+ * @param order_by Input column to generate ranks for.
+ * @param stream CUDA stream used for device memory operations and kernel launches.
+ * @param mr Device memory resource used to allocate the returned column's device memory.
+ * @return rank values.
+ */
+std::unique_ptr<column> inclusive_percent_rank_scan(column_view const& order_by,
```

Comment:
Do we have a public API for this? Then we can use copydoc.

Reply:
No, I'm afraid we don't have a `cudf::inclusive_*rank_scan()`.